src/EditorFeatures/CSharpTest2/Recommendations/ClassKeywordRecommenderTests.cs

```diff
@@ -401,5 +401,12 @@ public async Task TestAfterRecord()
             await VerifyKeywordAsync(
 @"record $$");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestAfterFileScopedNamespace()
+        {
+            await VerifyKeywordAsync(
+@"namespace NS; $$");
```

Comment:
ideally a test of namespace keywords that shows it does *not* appear here.  Note: if it does, that's not a big deal, just would be something nice to clean up later.

We should also have tests for `using` appearing after and before single-line ns.

Reply:
Done